sig-testing/charter.md

```diff
@@ -0,0 +1,82 @@
+# SIG Testing Charter
+
+This charter adheres to the conventions described in the 
+[Kubernetes Charter README] and uses the Roles and Organization Management
+outlined in [sig-governance].
+
+## Scope
+
+SIG Testing is interested in effective testing of Kubernetes.  We do not write or
+troubleshoot the project's tests, but instead focus on tooling that makes it
```

Comment:
We often do troubleshoot though...

Reply:
It should be presented as an act of goodwill, though, and best-effort.

Reply:
I'm rewording to be a little more explicit that we should be the last ask, not the first ask, and our goal is remediation rather than owning a fix.